patch/llpcPatchInOutImportExport.cpp

```diff
@@ -3324,6 +3324,40 @@ Value* PatchInOutImportExport::PatchCsBuiltInInputImport(
             pInput = ConstantInt::get(m_pContext->Int32Ty(), numSubgroups);
             break;
         }
+    case BuiltInGlobalInvocationId:
+        {
+            IRBuilder<> builder(*m_pContext);
```

Comment:
Try not to use IR builder in this function. Make the codes consistent. I can accept use IR builder in separated function internally. If we use IR builder, all codes within this function use it. Otherwise, follow existing style.

Reply:
I separated the functions like in #199.